src/status_im2/contexts/onboarding/notifications/views.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.contexts.onboarding.notifications.views
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im.notifications.core :as notifications]
+    [status-im2.contexts.onboarding.notifications.style :as style]
+    [status-im2.contexts.onboarding.common.background :as background]))
+
+(defn page-title
+  []
+  [rn/view {:style style/title-container}
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/title-text-style)
+    (i18n/label :t/intro-wizard-title6)]
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/subtitle-text-style)
+    (i18n/label :t/enable-notifications-sub-title)]])
+
+(defn navigate-back
+  [root]
+  [rn/view {:margin-top (if platform/ios? 44 0)}
+   [quo/page-nav
+    (merge {:horizontal-description? false
+            :one-icon-align-left?    true
+            :align-mid?              false
+            :page-nav-color          :transparent
+            :mid-section             {:type            :text-with-description
+                                      :main-text       nil
```

Comment:
Quick question: it's often not necessary to pass nil values in maps. Is this because you're replacing the default value of `:main-text` & `:description-img` and need to force a nil?

Reply:
@ilmotta not really. The page-nav component did not really support passing the props without `:mid-section` the reason why I passed nil here but I have updated the component with a `when` clause to first check if mid-section prop has been passed or not hence this is not needed anymore. You have a fair point

Reply:
Thanks @jo-mut, that makes sense.